commons/base-os/push.sh

```diff
@@ -0,0 +1,3 @@
+docker pull cloudsuitetest/debian:amd64 
```

Comment:
Why is this first line needed?

Also, why do we need to have a separate script for pushing?
You can merge these two scripts and then just pass an argument to the script that indicates the push should happen or not.

Reply:
As you suggested having a unified interface throughout the jobs, I decided to use build-and-push action for base-os that required such synchronization to make sure all images for different archs are ready for manifesting and pushing. The push job runs on another machine so the first line pulls the image. Actually, in this file, we will add arm64 and riscv64 images to pull three images and manifest all of them as base-os. As ARM and RISCV support is not a part of this PR, in the current version you see only amd64 image.